src/transformStateWithClones.js

```diff
@@ -7,7 +7,39 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const stateHistory = [];
+  let stateCopy = { ...state };
+
+  for (const action of actions) {
+    stateCopy = { ...stateCopy };
```

Comment:
you can remove it on 11 line and spread here like `const stateCopy = { ...state };`